### PR TITLE
Cmdct 3752 - Deploy UI services

### PR DESCRIPTION
**Pull request by BearHanded**
### Description

Deploy a login page without login functionality. This is just deploying the services for ui and ui-src, which will get us our cloudfront distributions. We can use those to get real URLs.

What cool stuff does this do?
- Integrate Deployments with AWS
- Includes a bunch of our actions
- Takes a slimmed down version of MFP to just facilitate a login page. Keeps some ui framework for state and whatnot, but open to changing any of it.
- Sets up ssm params (all envs configured)

### Related ticket(s)

CMDCT-3752

---
### How to test

https://d24wcdycbgw2lz.cloudfront.net/ <- This exists at all

### Important updates

---
### Author checklist

- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/assets/logos/logo_mdct_mfp_footer.png
Comment (angelaco11) [inline services/ui-src/src/assets/logos/logo_mdct_mfp_footer.png:1]: just curious, do we need this mfp image?

**Review by ajaitasaini — APPROVED** [review]

File: services/ui-src/src/assets/logos/logo_mdct_mfp_footer.png
Comment (BearHanded) [reply services/ui-src/src/assets/logos/logo_mdct_mfp_footer.png:1]: we'll need an equivalent for hcbs, and we can swap it over

**Review by angelaco11 — APPROVED** [review]
I think this looks good! Thanks for walking me through the code earlier

File: services/ui-src/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta name="description" content="Medicaid Data Collection Tool" />
```
Comment (rocio-desantiago) [inline services/ui-src/index.html:7]: should `"description"` match `<title>` here?

File: services/ui-src/public/favicon-16x16.png
Comment (rocio-desantiago) [inline services/ui-src/public/favicon-16x16.png:1]: I suppose these should be updated in the future with HCBS icons

File: services/ui-src/src/assets/logos/logo_mdct_mfp.png
Comment (rocio-desantiago) [inline services/ui-src/src/assets/logos/logo_mdct_mfp.png:1]: Shall we delete this one?

File: services/ui-src/src/assets/logos/logo_mdct_mfp_footer.png
Comment (rocio-desantiago) [reply services/ui-src/src/assets/logos/logo_mdct_mfp_footer.png:1]: 👍

**Review by rocio-desantiago — COMMENTED** [review]
Missed the merge cutoff, but just following up on the favicon. Is there one being designed for HCBS?

**Pull request merged**